README.md

```diff
@@ -173,6 +173,38 @@ The Geolite2-City database can be downloaded from MaxMind's site at [https://dev
 Download the GeoIP2 Binary and save it at `geo_data/GeoLite2-City.mmdb`.
 The app will start using that Geolite2 file for geolocation after restart.
 
+#### Testing on a mobile device or in a virtual machine
+
+By default, the application binds to `localhost`. To test on a network device or within a virtual machine, you can bind to `0.0.0.0` instead, using the following instructions:
+
+1. Determine your computer's network IP address. On macOS, you can find this in the "Network" system settings, shown under the "Status: Connected" label. This often takes the format of `192.168.1.x` or `10.0.0.x`.
+2. In `config/application.yml`, replace `localhost` in the `domain_name` setting with the IP address discovered in the previous step. Leave the trailing port `:3000` unchanged.
```

Comment:
We have some similar info about local development for windows about updating the ports, do you think we should add that here? Or just leave in the handbook:

 https://handbook.login.gov/articles/windows-virtual-machine.html#configure-the-idp-for-local-development

Reply:
> We have some similar info about local development for windows about updating the ports, do you think we should add that here? Or just leave in the handbook:

Hm, I wasn't aware that this page exists. I think it could be useful to have a general guide about Internet Explorer testing in the handbook, since it extends beyond just the IDP, but for IDP-specific configuration and based on the fact that it wasn't obvious for me to look here, maybe that should be reconciled with the advice here.

I'm not familiar with the `10.0.2.2` IP advice there. I know VirtualBox has a bunch of different network settings, so I'll want to double-check what I've got. It'd certainly be easier to advise a single static value than to recommend going through Network settings to find the network IP address.

Reply:
I did find some more detail about the `10.0.2.2`, and it does work for me in VirtualBox.

I think it'd be easier to recommend that... but then, if someone wants to develop on a phone (or a VM that's not VirtualBox?), they'd still need to do the instructions as currently included here 🤷 

I'm thinking....

- Keep this as-is
- Update handbook article to point here for additional guidance for IDP

I'm open to alternatives though.

Reply:
Let's update the handbook to describe the new `HOST=abc make run` syntax and point here?

Reply:
> Let's update the handbook to use the new `HOST=abc make run` syntax and point here?

Sounds good 👍